db/migrate/20190520220022_create_order_items.rb

```diff
@@ -4,8 +4,8 @@ def change
       t.references :item, foreign_key: true
       t.references :order, foreign_key: true
       t.integer :quantity
-      t.decimal :price
```

Comment:
Why delete? Doesn't match project schema.

Reply:
Resolved on branch (changed name of "price" to "price_per_item" (instead of "quantity_price")